bundles/org.openhab.core.thing.xml/src/main/java/org/eclipse/smarthome/core/thing/xml/internal/ThingTypeXmlProvider.java

```diff
@@ -120,9 +120,9 @@ public synchronized void addingObject(List<?> types) {
     private void addConfigDescription(ConfigDescription configDescription) {
         if (configDescription != null) {
             try {
-                this.configDescriptionProvider.add(this.bundle, configDescription);
-            } catch (Exception ex) {
-                this.logger.error("Could not register ConfigDescription!", ex);
+                configDescriptionProvider.add(bundle, configDescription);
+            } catch (Exception e) {
```

Comment:
You must have expected this question: Can we catch a more specific exception?

Reply:
There aren't any exceptions defined so it's not exactly clear what can be thrown here. So I think it's better to only catch the `RuntimeExceptions` here such as IAE/NPEs etc.